tests/class_methods_new_line.py

```diff
@@ -0,0 +1,42 @@
+class ClassSimplest:
+    pass
+class ClassWithInit:
+    def __init__(self):
+        pass
+class ClassWithInitAndVars:
+    cls_var = 100
+    def __init__(self):
+        pass
+class ClassWithInitAndVarsAndDocstring:
+    """Test class"""
+    cls_var = 100
+    def __init__(self):
+        pass
+
+
+# output
+
+
+class ClassSimplest:
+    pass
+
+
+class ClassWithInit:
+
+    def __init__(self):
+        pass
+
+
+class ClassWithInitAndVars:
+    cls_var = 100
+
+    def __init__(self):
+        pass
+
+
+class ClassWithInitAndVarsAndDocstring:
+    """Test class"""
+    cls_var = 100
```

Comment:
@ambv, I'd usually put an empty line between docstrings and the rest of the code below. What's your view on this?

Reply:
Black currently does not enforce that. What does PEP 257 say?

Reply:
> There's no blank line either before or after the docstring.

> Insert a blank line after all docstrings (one-line or multi-line) that document a class -- generally speaking, the class's methods are separated from each other by a single blank line, and the docstring needs to be offset from the first method by a blank line.

(src: https://www.python.org/dev/peps/pep-0257/)